backend/compact-connect/lambdas/python/common/cc_common/data_model/client.py

```diff
@@ -44,7 +44,7 @@ def get_or_create_provider_id(self, *, compact: str, ssn: str) -> str:
         # This is an 'ask forgiveness' approach to provider id assignment:
         # Try to create a new provider, conditional on it not already existing
         try:
-            self.config.provider_table.put_item(
+            self.config.ssn_table.put_item(
                 Item={
                     'pk': f'{compact}#SSN#{ssn}',
                     'sk': f'{compact}#SSN#{ssn}',
```

Comment:
If the pk and sk are the same, what is the purpose of the inverted GSI? Is the GSI going to be needed for future ssn work?

Reply:
Yeah, I'm anticipating that we will update the key structure and use the inverted key so that we can support both exchanging ssn->provider_id as well as provider_id->ssn